mypy/test/testcmdline.py

```diff
@@ -115,7 +115,9 @@ def normalize_file_output(content: List[str], current_abs_path: str) -> List[str
     """Normalize file output for comparison."""
     timestamp_regex = re.compile(r'\d{10}')
     result = [x.replace(current_abs_path, '$PWD') for x in content]
-    result = [re.sub(r'\b' + re.escape(__version__) + r'\b', '$VERSION', x) for x in result]
+    version = mypy.version.__version__
+    result = [re.sub(r'\b' + re.escape(version) + r'\b', '$VERSION', x) for x in result]
+    base_version = getattr(mypy.version, 'base_version', version)
```

Comment:
When would `base_version` ever be missing? Or is this another hack to fool mypyc?

Reply:
`base_version` is missing in all installed versions of mypy, due to https://github.com/python/mypy/blob/master/setup.py#L56-L61

Reply:
Ah... That's worth a comment here.